templates/Admin/Translate/index.php

```diff
@@ -67,7 +67,7 @@
 <?php } else { ?>
 
 	<p>
-<?php echo __d('translate', 'Please create a project first'); ?>: <?php echo $this->Html->link('Project Index', ['prefix' => 'admin', 'controller' => 'TranslateProjects']); ?>
+<?php echo __d('translate', 'Please create a project first'); ?>: <?php echo $this->Html->link('Project Index', ['controller' => 'TranslateProjects']); ?>
```

Comment:
```suggestion
<?php echo __d('translate', 'Please create a project first'); ?>: <?php echo $this->Html->link('Project Index', ['prefix' => 'Admin', 'controller' => 'TranslateProjects']); ?>
```

shouldnt it be still an Admin link?

Reply:
In that case, the lines above and below these changes would need to be adjusted as well. But as soon as you add the prefix, the router is complaining it can't find a routing scheme for it. When you remove the prefix from the routing link, for some reason the link does not throw a routing error and the link directs to the admin/translate/translate-projects location.

Reply:
I guess either you add the prefix **and** the plugin, or you omit both. In latter case, Cake would inherit the current situation where the prefix is already set to admin since we are in admin and the plugin is already set to translate since this is a template for the translate plugin.